moment.parseFormat.js

```diff
@@ -95,17 +100,17 @@
     format = format.replace(regexTimezone, 'Z');
 
     // 05:30:20pm ☛ hh:mm:ssa
-    format = format.replace(regexHoursWithLeadingZeroDigitMinutesSecondsAmPm, 'hh:mm:ss$1a');
+    format = format.replace(regexHoursWithLeadingZeroDigitMinutesSecondsAmPm, appendAMPMCase(format, 'hh:mm:ss$1'));
```

Comment:
I wonder if we could find a way to not run the `appendAMPMCase` 6 times. Maybe we could set `amOrPm` in lie 36 to lowercase / uppercase, depending wether `(AM|PM)` or `(am|pm)` matches?